tracer/src/Datadog.Trace/AgentProcessManager.cs

```diff
@@ -69,8 +68,9 @@ public static void Initialize()
                 var azureAppServiceSettings = new ImmutableAzureAppServiceSettings(GlobalConfigurationSource.Instance);
 
                 var automaticTraceEnabled = EnvironmentHelpers.GetEnvironmentVariable(ConfigurationKeys.TraceEnabled, string.Empty)?.ToBoolean() ?? true;
+                var automaticProfilingEnabled = EnvironmentHelpers.GetEnvironmentVariable(ContinuousProfiler.ConfigurationKeys.ProfilingEnabled)?.ToBoolean() ?? false;
```

Comment:
> ?? false

:oh: I thought it was enabled by default now.

Reply:
yep profiler is disabled by default